theme/less/_navigation.less

```diff
@@ -1,26 +1,67 @@
-nav
+header nav
 {
-  ul
+  border-top: 1px solid #E9E9E9;
+  box-shadow: 0px 8px 15px rgba(0, 0, 0, 0.06);
+}
+
+header nav ul
+{
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+header nav ul a
+{
+  display: block;
+  font-size: 24px;
+  text-decoration: none;
+  color: @primary;
+  padding: 25px;
+}
+
+#hamburger
+{
+  background-color: transparent;
+  cursor: pointer;
+  border: 0;
+  background-image: url(/images/hamburger.svg);
+  width: 32px;
+  height: auto;
+}
+
+#hamburger-menu
+{
+  text-align: center;
+  overflow: hidden;
+  transition: height 1s ease-in-out;
```

Comment:
Suggestion: .5s feels more natural to me, what you think? 1s feels too slow to me.

Reply:
Done.